python-backend/app/api/document_manager/resources/document_manager.py

```diff
@@ -60,20 +68,18 @@ def post(self):
         pretty_folder = data.get('pretty_folder')
         pretty_path = os.path.join(base_folder, pretty_folder, filename)
 
-        try:
-            if not os.path.exists(folder):
-                os.makedirs(folder)
-            with open(file_path, "wb") as f:
-                f.seek(file_size -1)
-                f.write(b"\0")
-        except IOError as e:
-            return self.create_error_payload(500, 'Unable to create file'), 500
+        #try:
+        if not os.path.exists(folder):
+            os.makedirs(folder)
+        with open(file_path, "wb") as f:
+            f.seek(file_size - 1)
+            f.write(b"\0")
+        #except IOError as e:
+        #    return self.create_error_payload(500, 'Unable to create file'), 500
```

Comment:
If this `try` is being removed then it can just be removed. What's the reason for commenting it out?

Reply:
Done for troubleshooting but missed reverting the change. Reverted.